packages/fluxible-router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluxible-router",
-  "version": "2.0.0-beta.3.0",
+  "version": "2.0.0-beta.4.0",
```

Comment:
FYI this change will not make it into 1.x branch, which we use internally. If you need it for existing apps, you may need to backport this change to 1.x.